### DISCUSSION TITLE
High CPU Utilization in the database sever

### DISCUSSION THREAD

**Discussion by engahmedtahaa** [jobrunr/jobrunr#649] [Q&A]
We’ve been using JobRunr scheduled jobs for a while with a SQL server 2019 database and we are using jobrunr latest version

we figure out very strange behavior in the database which takes High CPU Utilization Random along the day, investigation in the top used to query the below one is the most CPU usage query 

(@P0 bigint,@P1 int)select * from jobrunr_jobs_stats ORDER BY total ASC OFFSET @P0 ROWS FETCH NEXT @P1 ROWS ONLY

can anyone support us in this issue we did not know what is the problem and there are no errors or exceptions appearing

**Comment by rdehuyss** [comment]
If you want commercial support, please contact us for a support contract.

**Comment by rdehuyss** [comment]
Cause found: a lot of failed jobs resulting in really old jobs not being cleaned up.
